packages/internal/src/generate/graphqlCodeGen.ts

```diff
@@ -133,23 +134,63 @@ function getLoadDocumentsOptions(filename: string) {
   return loadTypedefsConfig
 }
 
+function getPrismaClient(hasGenerated = false): any {
+  const localPrisma = require('@prisma/client')
+
+  if (!localPrisma.ModelName) {
+    if (hasGenerated) {
+      return {}
+    } else {
+      if (process.env.NODE_ENV === 'test') {
+        // When running as part of tests `yarn rw` isn't available as a bin
+        // In actual projects it will be available
+        return {}
+      }
+
+      execa.sync('yarn rw prisma generate', { shell: true, stdio: 'inherit' })
```

Comment:
Thoughts about not inheriting here? I know we need a better set of best practices around this, but It cuts up the `Listr` input:

```
~/p/test-project (main) [1]> yarn rw g sdl userExample
  ✔ Generating SDL files...
    ✔ Successfully wrote file `./api/src/graphql/userExamples.sdl.ts`
    ✔ Successfully wrote file `./api/src/services/userExamples/userExamples.scenarios.ts`
    ✔ Successfully wrote file `./api/src/services/userExamples/userExamples.test.ts`
    ✔ Successfully wrote file `./api/src/services/userExamples/userExamples.ts`
  ⠙ Generating types ...

Running Prisma CLI...
$ yarn prisma generate --schema /Users/dom/prjcts/test-project/api/db/schema.prisma

Environment variables loaded from .env
Prisma schema loaded from api/db/schema.prisma

  ✔ Generating SDL files...
    ✔ Successfully wrote file `./api/src/graphql/userExamples.sdl.ts`
    ✔ Successfully wrote file `./api/src/services/userExamples/userExamples.scenarios.ts`
    ✔ Successfully wrote file `./api/src/services/userExamples/userExamples.test.ts`
    ✔ Successfully wrote file `./api/src/services/userExamples/userExamples.ts`
  ✔ Generating types ...
```

Reply:
![image](https://user-images.githubusercontent.com/30793/187035441-590f9768-60a7-4aa9-93ee-401786d044bf.png)